### PR TITLE
added gem bullet and changes in config/envir/development.rb

### PR DESCRIPTION
**Pull request by kasiaekiert**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -67,6 +67,7 @@ group :development do
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
   gem 'pry-rails'
+  gem 'bullet'
```
[the remark this answers is not in the document]
Comment (womanonrails) [reply Gemfile]: I agrree with that

**Comment by kasiaekiert** [comment]
May I do pull request? Or we will do sth here? 

 Includes method??

File: Gemfile.lock [hunk 2]
```diff
@@ -215,8 +218,8 @@ GEM
       rainbow (>= 2.2.2, < 4.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 1.6)
-    rubocop-performance (1.3.0)
-      rubocop (>= 0.68.0)
+    rubocop-performance (0.0.1)
```
Comment (womanonrails) [inline Gemfile.lock]: When you add only `bullet` gem you cannot have in `Gemfile.lock` changes for `rubocop-performance` or this two gems are dependent. Please run one more time `bundle` command on this branch and check.

File: Gemfile [hunk 3]
```diff
@@ -59,9 +59,12 @@ group :development, :test do
 end
 
 group :development do
+  gem 'bullet'
+
   # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
   gem 'web-console', '>= 3.3.0'
   gem 'listen', '>= 3.0.5', '< 3.2'
+
```
[the remark this answers is not in the document]
Comment (womanonrails) [reply Gemfile]: You have here additional line.

**Review by womanonrails — CHANGES_REQUESTED** [review]

**Review by womanonrails — APPROVED** [review]

**Pull request merged**